CompilerSource/settings.h

```diff
@@ -72,7 +70,25 @@ namespace setting
   extern bool inherit_objects;  // Determines whether objects should automatically inherit locals and events from their parents
   extern bool automatic_semicolons; // Determines whether semicolons should automatically be added or if the user wants strict syntax
   extern COMPLIANCE_LVL compliance_mode; // How to resolve differences between GM versions.
-  extern string keyword_blacklist; //Words to blacklist from user scripts, separated by commas.
+  extern std::string keyword_blacklist; //Words to blacklist from user scripts, separated by commas.
 }
 
+struct CompilerInfo {
+  std::string name;
+  std::string maintainer;
+  std::string target_platform;
+
+  std::map<std::string, std::string> parser_vars;
```

Comment:
Okay, fine, but can you include the list of supported tags in a comment, if you're not going to declare a variable for each? We don't have a schema for these (eg, a protobuf or general description of supported names).